### PR TITLE
Add account_name attribute

### PR DESCRIPTION
**Pull request by LiamSennitt**
- Add `account_name` attribute
- Fix `account_uid` description formatting

### CODE REVIEW

File: objects/resource.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "extends": "object",
   "attributes": {
     "account_uid":{
-      "description": "The unique identifier of the resource owning account(e.g. AWS Account ID).",
+      "description": "The unique identifier of the resource owning account (e.g. AWS Account ID).",
```
Comment (rroupski) [inline objects/resource.json:8]: What does the description mean? The resource owns the account or the account owns the resource?

**Comment by rroupski** [comment]
Please update the version number.

**Review by rroupski — CHANGES_REQUESTED** [review]
Update the version number.

File: objects/resource.json (hunk 1 above)
Comment (floydtree) [reply objects/resource.json:8]: I interpret it as, account that owns the resource.

Comment (LiamSennitt) [reply objects/resource.json:8]: I just added the missing space 😅

**Review by floydtree — APPROVED** [review]

File: dictionary.json [hunk 2]
```diff
@@ -67,7 +67,12 @@
     },
     "account_uid": {
       "caption": "Account UID",
-      "description": "The unique identifier of the account(e.g. AWS Account ID).",
+      "description": "The unique identifier of the account (e.g. AWS Account ID).",
+      "type": "string_t"
+    },
+    "account_name": {
+      "caption": "Account Name",
+      "description": "The unique identifier of the account (e.g. AWS Account Name).",
```
Comment (LiamSennitt) [inline dictionary.json:75]: This should be "The name of...", I'll update that 👍

File: objects/resource.json (hunk 1 above)
Comment (LiamSennitt) [reply objects/resource.json:8]: Out of interest could I remove this description as it's duplicated in the `dictionary.json`?

**Review by rroupski — CHANGES_REQUESTED** [review]
Since you are changing the resource.json file, can you update the `account_uid` description as: `The unique identifier of the account that owns the resource (e.g. AWS Account ID).`

**Review by rroupski — APPROVED** [review]

**Pull request merged**